test/parallel/test-runner-output.mjs

```diff
@@ -103,15 +128,16 @@ const tests = [
   process.features.inspector ? { name: 'test-runner/output/coverage_failure.js' } : false,
 ]
 .filter(Boolean)
-.map(({ name, tty, transform }) => ({
+.map(({ name, tty, transform, options }) => ({
   name,
   fn: common.mustCall(async () => {
     await snapshot.spawnAndAssert(fixtures.path(name), transform ?? defaultTransform, { tty });
   }),
+  options
 }));
 
 describe('test runner output', { concurrency: true }, () => {
-  for (const { name, fn } of tests) {
-    it(name, fn);
+  for (const { name, fn, options } of tests) {
+    it(name, fn, options);
```

Comment:
I think the order should be:
```suggestion
    it(name, options, fn);
```

https://github.com/nodejs/node/blob/01719333623aaa324d05011ffcba0dddc0ea3666/doc/api/test.md#L935
https://github.com/nodejs/node/blob/01719333623aaa324d05011ffcba0dddc0ea3666/lib/internal/test_runner/test.js#L423-L425